packages/core/src/link-refs.ts

```diff
@@ -0,0 +1,38 @@
+import { SolcBytecode } from './solc-api';
+
+export interface LinkReference {
+  src: string;
+  name: string;
+  length: number;
+  start: number;
+  placeHolder: string;
```

Comment:
This can be capitalized as `placeholder`.

Reply:
Agree. Will do.